src/command/ig.jl

```diff
@@ -686,23 +686,16 @@ end
 
 "Plot a simple price chart"
 function ig_chart(symbol::AbstractString, dates::Vector{Date}, values::Vector{<:Real})
-    theme(:dark)
-    height = 720
-    width = height * 16 ÷ 9
     from_date, to_date = extrema(dates)
     last_price_str = format_amount(last(values))
-    p = plot(dates, values,
-        title = "$symbol Historical Prices ($from_date to $to_date)\nLast price: $last_price_str",
-        linewidth = 2,
-        size = (width, height),
-        legend = nothing,
-        tickfontsize = 12,
-    )
+    c = crplot(dates, values; xticks = 5, yticks = 10, title="$symbol Historical Prices ($from_date to $to_date)\nLast price: $last_price_str")
```

Comment:
You can change the `width` and the `height` too, which are the respective keywords in `crplot` (don't have `size` yet). I will add more keywords later.

Reply:
We can make changes to that in the following days, I think.

Reply:
yeah, it was mostly to let you know.